build.gradle

```diff
@@ -27,6 +27,10 @@ reckon {
 
 repositories {
     mavenLocal()
+    maven {
+        name = 'modrinth'
+		url = "https://api.modrinth.com/maven"
+	}
```

Comment:
Indendation looks a bit messed up here. Can you fix this before I metge?

Reply:
sorry for taking so long I was busy